include/unifex/scheduler_concepts.hpp

```diff
@@ -97,6 +97,43 @@ inline constexpr struct schedule_after_cpo {
       -> decltype(tag_invoke(*this, (TimeScheduler &&) s, (Duration &&) d)) {
     return tag_invoke(*this, (TimeScheduler &&) s, (Duration &&) d);
   }
+
+  template<typename Duration>
+  class schedule_after_sender {
+  public:
+    template<template<typename...> class Variant, template<typename...> class Tuple>
+    using value_types = Variant<Tuple<>>;
+
+    template<template<typename...> class Variant>
+    using error_types = Variant<std::exception_ptr>;
+
+    explicit schedule_after_sender(Duration d)
+    : duration_(d)
+    {}
+
+  private:
+    friend schedule_after_cpo;
+
+    template<typename Receiver>
+    friend auto tag_invoke(tag_t<connect>, const schedule_after_sender& s, Receiver&& r)
+    -> std::invoke_result_t<
+          decltype(connect),
+          std::invoke_result_t<
+              schedule_after_cpo,
+              std::invoke_result_t<decltype(get_scheduler), const Receiver &>,
+              const Duration&>,
+          Receiver> {
```

Comment:
Style nit: this return type is hella long. Consider making an alias template for it.

Reply:
This should be improved now.